conjur/cli.py

```diff
@@ -59,143 +63,154 @@ def copyright():
+                                                             help='Get the value of a variable')
+        set_variable_parser = variable_subparsers.add_parser('set',
+                                                             help='Set the value of a variable')
+
+        get_variable_parser.add_argument('variable_id',
+                                         help='ID of a variable', nargs='+')
+
+        set_variable_parser.add_argument('variable_id',
+                                         help='ID of the variable')
+        set_variable_parser.add_argument('value',
+                                         help='New value of the variable')
+
+        resource_subparsers.add_parser('whoami',
+                                       help='Provides information about the current logged-in user')
 
         global_optional.add_argument('-h', '--help', action='help', help="Display help list")
         global_optional.add_argument('-v', '--version', action='version',
                                      help="Display version number",
-                                     version='Conjur Python CLI version ' + __version__ + "\n"
+                                     version='Conjur CLI version ' + __version__ + "\n"
                                              + self.copyright())
 
-        global_optional.add_argument('-l', '--url')
-        global_optional.add_argument('-a', '--account')
-
-        global_optional.add_argument('-c', '--ca-bundle')
-        global_optional.add_argument('--insecure',
-            help='Skip verification of server certificate (not recommended)',
-            dest='ssl_verify',
-            action='store_false')
-
-        # The external names for these are unfortunately named so we remap them
-        global_optional.add_argument('-u', '--user', dest='login_id')
-        global_optional.add_argument('-k', '--api-key')
-        global_optional.add_argument('-p', '--password')
-
         global_optional.add_argument('-d', '--debug',
-            help='Enable debugging output',
-            action='store_true')
-
-        global_optional.add_argument('--verbose',
-            help='Enable verbose debugging output',
-            action='store_true')
+                                     help='Enable debugging output',
+                                     action='store_true')
 
+        global_optional.add_argument('--insecure',
+                                     help='Skip verification of server certificate (not recommended for production)',
+                                     dest='ssl_verify',
+                                     action='store_false')
 
         resource, args = Cli._parse_args(parser)
 
-        # We don't have a good "debug" vs "verbose" separation right now
-        if args.verbose is True:
-            args.debug = True
-
         # TODO Add tests for exception handling logic
+        # # pylint: disable=broad-except
         try:
+            self._setup_logging(args.debug)
```

Comment:
Is failure in setting up the logger expected? if not, it may need to be moved out of the `try/except`